types/cie-rgb-color-converter/package.json

```
@@ -0,0 +1,12 @@
+{
+    "private": true,
+    "name": "@types/cie-rgb-color-converter",
+    "version": "1.0.9999",
+    "projects": [
+        "https://github.com/Shnoo/js-CIE-1931-rgb-color-converter#readme"
+    ],
+    "devDependencies": {
+        "@types/cie-rgb-color-converter": "workspace:."
+    },
+    "owners": []
```

```suggestion
    "owners": []
```
You should add a owner